ngraph_bridge/ngraph_builder.cc

```diff
@@ -913,9 +913,37 @@ static Status TranslateBatchMatMulOp(
   return Status::OK();
 }
 
-static Status TranslateBiasAddOp(const Node* op,
-                                 const std::vector<const Tensor*>&,
-                                 Builder::OpMap& ng_op_map) {
+static Status TranslateBatchMatMulV2Op(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_lhs, ng_rhs;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_lhs, &ng_rhs));
+
+  auto ng_lhs_shape = ng_lhs->get_shape();
+  auto ng_rhs_shape = ng_rhs->get_shape();
+  size_t n_dims = ng_lhs_shape.size();
+
+  for (size_t i = 0; i < n_dims - 2; ++i) {
```

Comment:
Is this check also applicable for `TranslateBatchMatMulOp` In which case we can just add the check in `TranslateBatchMatMulOp` and use the same function for `BatchMatMul` and `BatchMatMulV2`

Reply:
I was thinking about maintainability when more features will be needed in `BatchMatMulV2`, it will be easier to add them only changing according to translate function, without taking into account old `BatchMatMul` version. What's more when in code someone makes a mistake by using old `BatchMatMul` instead of `BatchMatMulV2` it could be working having only one translation function. I think such behavior can lead to confusion and complicate the debugging of the model.

Reply:
@maxbubula Are we supporting the entire functionality of BatchMatMulV2 TF op or what is required for BERT alone?

Reply:
We are supporting only the part of functionality required for BERT alone. I think that BatchMatMul (V1) also do not implement all the functionalities.

Reply:
@avijit-nervana  Is this acceptable? BatchMatMulV2 will support only the functionality for BERT.

Reply:
Checked with @avijit-nervana  we can focus on just BERT functionality for now